00-HTML-CSS-basics/index.html

```diff
@@ -0,0 +1,101 @@
+              <!--LUCIA PUCCINI Topic 0 - HTML & CSS Basics-->
+              <!--copied from example-->
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <title>Index</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link rel="manifest" href="site.webmanifest">
+  <link rel="apple-touch-icon" href="icon.png">
+  <!-- Place favicon.ico in the root directory -->
+
+ <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+  <!--[if lte IE 9]>
+    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+  <![endif]-->
+
+<header class="content">
+  <h1>Header</h1>
+</header>
+
+<div class="column-layout">
+
+  <div class="sidebar1">
+
+    <nav class="content">
+      <h2>Nav bar</h2>
+      <br><br><br><br><br>
+    </nav>
+
+  </div>
+
+  <div class="main-column">
+
+      <section class="content">
+
+        <h1>Section</h1>
+        <div class="">
+          <header class="small-content">
+            <h2> Section header  </h2>
+          </header>
+        </div>
+        <div class="">
+            <article class="small-content">
+              <h2>Section article</h2>
+            </article>
+        </div>
+        <div class="">
+          <footer class="small-content">
+          <h2>Section footer</h2>
+          </footer>
+        </div>
+        </section>
+
+    </div>
+
+    <div class="sidebar2">
+
+      <aside class="content">
+        <h2>Aside </h2>
+        <br><br><br><br><br>
+      </aside>
+
+    </div>
+
+</div>
+
+
+
+
+  <!-- Add your site or application content here -->
+  <p></p>
+  <script src="js/vendor/modernizr-{{MODERNIZR_VERSION}}.min.js"></script>
+  <script src="https://code.jquery.com/jquery-{{JQUERY_VERSION}}.min.js" integrity="{{JQUERY_SRI_HASH}}" crossorigin="anonymous"></script>
```

Comment:
This is quick akward. Is right to have it this way? Why is this here? What is this used for?